### ISSUE TITLE
Accessibility: Missing alt text for image

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#6]
CAG 2.0 Success Criteria 1.1.1 - Non text Content Level (A) Compliance violation - All images must have equivalent alternative text.
1. Go to: data.ottawa.ca
2. Observe for alt text on the image associated with the 'Map Quest' icon at the bottom of the map images.

Resolution: Add alternative text to images.
![mapquest icon_alt text missing](https://cloud.githubusercontent.com/assets/5271253/8012587/c2709096-0b90-11e5-9dd4-27622dbbf57b.JPG)
